### ISSUE TITLE
preprocessing and "refresh" actions

### ISSUE THREAD

**Issue by annalisapg** [gama-platform/gama#1259]
Hi all,
another enhancement that could be nice to have implemented in GAMA concerns the preprocessing of geographical data with the refresh of the working zone.
In fact, suppose I want to choose a specific geographic zone to execute my simulation and I can choose it only after have visualized it. It should be nice to do this thing in GAMA and start simulation (cycle 1) only after the resize/zoom of the working zone.
So, maybe a "refresh" option in the code (which is different from a "start simulation" command) could be a good answer to this issue.

Is this enhancement possible? do you think it could be useful?

**Comment by sriramab** [comment]
May be this link answers part of it?
http://code.google.com/p/gama-platform/issues/detail?id=884

you can look at viewport, or focus operators(?)

regards,
Srirama

**Comment by AlexisDrogoul** [comment]
I come back to this old request, and I must say that I don't fully understand it. There are many ways to specify dynamic "zones" (with focus: and zoom: facets in displays)... But maybe it is not exactly what you intend to do. Is the "refresh" you're talking about intended to refresh the displays only or are you talking about the simulation itself ? I close it for now on, feel free to reopen and specify a little bit more clearly the scenario you had in mind (or a specific use-case).
